src/main/java/org/candlepin/subscriptions/tally/UsageSnapshotProducer.java

```diff
@@ -86,31 +61,8 @@ public UsageSnapshotProducer(ProductIdToProductsMapSource productIdToProductsMap
     }
 
     @Transactional
-    @Timed("rhsm-subscriptions.snapshots.single")
-    public void produceSnapshotsForAccount(String account) {
-        produceSnapshotsForAccounts(Collections.singletonList(account));
-    }
-
-    @Transactional
-    @Timed("rhsm-subscriptions.snapshots.collection")
-    public void produceSnapshotsForAccounts(List<String> accounts) {
-        log.info("Producing snapshots for {} accounts.", accounts.size());
-        // Account list could be large. Only print them when debugging.
-        if (log.isDebugEnabled()) {
-            log.debug("Producing snapshots for accounts: {}", String.join(",", accounts));
-        }
-
-        Collection<AccountUsageCalculation> accountCalcs;
-        try {
-            accountCalcs = retryTemplate.execute(context ->
-                accountUsageCollector.collect(this.applicableProducts, accounts)
-            );
-        }
-        catch (Exception e) {
-            log.error("Could not collect existing usage snapshots for accounts {}", accounts, e);
-            return;
-        }
-
+    public void produceSnapshotsForAccounts(Collection<String> accounts,
```

Comment:
I think this should be renamed now, maybe `produceSnapshotsForAccountsFromCalculations`? I was initially a little confused at the parameter change, and I think a name change could help.

Reply:
DONE - I went with produceSnapshotsFromCalculations since "FromAccounts" was kinda implied by the parameter list.

Available in next commit.